crates/squawk_github/src/lib.rs

```diff
@@ -69,16 +69,18 @@ pub fn comment_on_pr(
     repo: &str,
     issue: i64,
     body: &str,
+    existing_comment_text_includes: &str,
 ) -> Result<(), GithubError> {
     let comments = gh.list_issue_comments(owner, repo, issue)?;
 
     let bot_name = gh.app_slug();
 
     info!("checking for existing comment");
-    match comments
-        .iter()
-        .find(|x| x.user.r#type == "Bot" && x.user.login == bot_name)
-    {
+    match comments.iter().find(|x| {
+        x.user.r#type == "Bot"
+            && x.user.login == bot_name
+            && x.body.contains(existing_comment_text_includes)
```

Comment:
I thought about using `starts_with` here to be even more specific, but I wasn't sure how that would play with the markdown generation

Reply:
ah is the issue that there's one bot account setup and multiple apps, other than squawk, use it?

Reply:
can we add a comment explaining the reasoning? otherwise looks good!

Reply:
Added!